tests/app/dao/test_notification_dao.py

```diff
@@ -568,7 +600,7 @@ def test_should_not_delete_sent_notifications_before_one_day(notify_db, notify_d
 
 def test_should_not_delete_failed_notifications_before_seven_days(notify_db, notify_db_session):
     expired = datetime.utcnow() - timedelta(hours=24 * 7)
-    valid = datetime.utcnow() - timedelta(hours=(24 * 6) + 23,  minutes=59, seconds=59)
+    valid = datetime.utcnow() - timedelta(hours=(24 * 6) + 23, minutes=59, seconds=59)
```

Comment:
Possibly one for the future reference if we do more and more datetime handling.
http://dateutil.readthedocs.org/en/latest/index.html